docs/configuration/index-config.md

```diff
@@ -125,7 +125,7 @@ fieldnorms: true
 | ------------- | ------------- | ------------- |
 | `description` | Optional description for the field. | `None` |
 | `stored`    | Whether value is stored in the document store | `true` |
-| `tokenizer` | Name of the `Tokenizer`, choices between `raw`, `default` and `en_stem` | `default` |
+| `tokenizer` | Name of the `Tokenizer`, choices between `raw`, `default`, `en_stem` and `chinese` | `default` |
```

Comment:
you even thought of updating the doc :)

Reply:
I wonder if we should rename it with something emphasize that this is not a real chinese tokenizer...

`chinese_rover`, `chinese_friendly` or whatever your creativity could come up with.